tests/test_torchinductor.py

```diff
@@ -1891,6 +1891,15 @@ def fn(a, b, c):
             ],
         )
 
+    @patch.object(config.triton, "max_tiles", 2)
+    def test_fuse_tiled(self):
+        def fn(a, b, c):
+            return a + b, c + 1
+
+        self.common(
```

Comment:
Similar to my comment about `test_vertical_fusion1`, it will be great if we can offer a way to check the generated code.